cli/atlas/main.go

```diff
@@ -227,11 +226,24 @@ func generateProtobuf() error {
 func resolveImports(dirs []string) error {
 	fmt.Print("Resolving imports... ")
 	for _, dir := range dirs {
-		err := exec.Command("goimports", "-w", dir).Run()
-		if err != nil {
+		if err := runCommand("goimports", "-w", dir); err != nil {
 			return err
 		}
 	}
 	fmt.Println("done!")
 	return nil
 }
+
+func runCommand(command string, args ...string) error {
+	cmd := exec.Command(command, args...)
+	stderr := &bytes.Buffer{}
+	cmd.Stderr = stderr
```

Comment:
feature request: capture `stdout` as well and print it if a -debug flag is present :)

Reply:
Nice idea! I'll add that.